src/søknad/steg/4-barnasbosted/BarnasBosted.tsx

```diff
@@ -0,0 +1,215 @@
+
+const BarnasBosted: React.FC = () => {
+  const intl = useIntl();
+  const { søknad } = useSøknadContext();
+  const [forelder, settForelder] = useState<IForelder>({
+    navn: "",
+    fødselsdato: null,
+    personnr: "",
+    borINorge: undefined,
+    avtaleOmDeltBosted: undefined,
+    harAnnenForelderSamværMedBarn: "",
+    harDereSkriftligSamværsavtale: "",
+    hvordanPraktiseresSamværet: "",
+    borISammeHus: "",
+    boddSammenFør: undefined,
+    flyttetFra: null,
+    hvorMyeSammen: ""
+  });
+
+  const location = useLocation();
+  const nesteRoute: IRoute = hentNesteRoute(Routes, location.pathname);
+  const forrigeRoute: IRoute = hentForrigeRoute(Routes, location.pathname);
+
+  const visSamværsavtaleAdvarsel = (valgtSvar: string) => {
+    return valgtSvar == intl.formatMessage({ id: 'barnasbosted.spm.jaIkkeKonkreteTidspunkt' });
+  }
+
+  const visSkriftligSamværsavtaleSpørsmål = (svarAndreForelderenSamvær: string) => {
+    return svarAndreForelderenSamvær && svarAndreForelderenSamvær !== intl.formatMessage({ id: 'barnasbosted.spm.andreForelderenSamværNei' });
+  }
+
+  const visHvordanPraktiseresSamværet = (valgtSamværsrett: string) => {
+    return valgtSamværsrett == intl.formatMessage({ id: 'barnasbosted.spm.jaIkkeKonkreteTidspunkt' });
+  }
+
+  const barn = søknad.person.barn[0];
+
+  const bosted = barn.harSammeAdresse ? intl.formatMessage({ id: 'barnekort.adresse.registrert' }) : intl.formatMessage({ id: 'barnekort.adresse.uregistrert' });
+
+  return (
+    <>
+      <Side
+        tittel={intl.formatMessage({ id: 'barnasbosted.sidetittel' })}
+        nestePath={nesteRoute.path}
+        tilbakePath={forrigeRoute.path}
+      >
+        <div className="barnas-bosted">
+          <div className="barnas-bosted__header">
+            <img alt="barn" className="barneikon" src={barn1} />
+          </div>
+          <div className="barnas-bosted__info">
+            <Element className="navn">{barn.navn}</Element>
+            <div className="inforad">
+              <div className="informasjonselement">
+                  <Normaltekst className="informasjonselement__header">{intl.formatMessage({id: 'barnekort.fødselsnummer'})}</Normaltekst>
+                  <Normaltekst className="informasjonselement__innhold">{barn.fnr}</Normaltekst>
+              </div>
+              <div className="informasjonselement">
+                  <Normaltekst className="informasjonselement__header">{intl.formatMessage({id: 'barnekort.alder'})}}</Normaltekst>
+                  <Normaltekst className="informasjonselement__innhold">{barn.alder} år</Normaltekst>
+              </div>
+              <div className="informasjonselement">
+                  <Normaltekst className="informasjonselement__header">{intl.formatMessage({id: 'barnekort.bosted'})}}</Normaltekst>
+                  <Normaltekst className="informasjonselement__innhold">{bosted}</Normaltekst>
+              </div>
+            </div>
+          </div>
```

Comment:
Tror denne kodebiten er perfekt for å trekkes ut i sin egen komponent, og kanskje perfekt for å teste ut `styled-components på `også!  Hvis du er gira på å teste det ut, så kan du ta en titt på `TilbakeKnapp.tsx` ✨ 

Da minsker du styling kode i BarnasBosted.less - fil, får samlet det et sted, og det blir litt lettere å skjønne hva BarnasBosted gjør :)

Reply:
Jess, skrev om til å bruke styled-components